TestFoundation/TestNSString.swift

```diff
@@ -401,7 +401,7 @@ class TestNSString : XCTestCase {
         var buf : [UInt8] = []
         buf.reserveCapacity(reservedLength)
         var usedLen : CFIndex = 0
-        buf.withUnsafeMutableBufferPointer { p in
+        _ = buf.withUnsafeMutableBufferPointer { p in
```

Comment:
Hmm, I think that I agree with you.  I will adjust the patch and upload it again.  Good thought!